releases.md

```diff
@@ -17,6 +21,13 @@ Examples:
 - A project may release every four months, and maintain a release branch for the three most recent minor
   releases
 
+Individual projects may provide short support windows for non-LTS major or minor releases, which may last
+until the next major or minor release is available.
+
+Note that the support policy is independent from the any API stability policy provided by projects.
+If a release by mistake broke an API stability policy, during the support period that would justify
+the creation a patch release to resolve the issue.
```

Comment:
@wlynch I updated the policy - let me know if it looks ok

I think deprecations and support are pretty independent.
Deprecation is effectively immediately once it's declared.
However a feature is deprecated and we discover a CVE in the code associated with it, we will still patch it until the EOL of that release because the code is part of the Tekton release, even when deprecated.